mongo/options/createcollectionoptions.go

```diff
@@ -26,6 +26,45 @@ func (d *DefaultIndexOptions) SetStorageEngine(storageEngine interface{}) *Defau
 	return d
 }
 
+// TimeSeriesOptions specifies options on a time-series collection.
+type TimeSeriesOptions struct {
+	// Name of the top-level field to be used for time. Inserted documents must have this field,
+	// and the field must be of the BSON UTC datetime type (0x9).
+	TimeField string
+
+	// Optional name of the top-level field describing the series. This field is used to group
+	// related data and may be of any BSON type, except for array. This name may not be the same
+	// as the timeField or _id.
```

Comment:
If this is referring to the Go option, I think the casing should match:
```suggestion
	// as the TimeField or _id.
```